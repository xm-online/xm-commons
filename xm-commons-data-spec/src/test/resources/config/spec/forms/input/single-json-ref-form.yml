test-items:
    - key: team/CREATE_EMPLOYEE
      inputForm: |
          {
            "form": [
              {
                "$ref": "#/xmForm/employeeForm"
              }
            ],
            "prop": {
              "hideNameAndDescription": true
            }
          }

forms:
    - key: employeeForm
      ref: json-forms/employeeForm.json
