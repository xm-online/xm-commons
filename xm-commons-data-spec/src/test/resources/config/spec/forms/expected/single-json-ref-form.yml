test-items:
    - key: team/CREATE_EMPLOYEE
      inputForm: |
          {
            "form": [
              { "key": "employeeName", "title": { "en": "First name" }, "type": "string" },
              { "key": "employeeLastName", "title": { "en": "Last name" }, "type": "string" },
              { "key": "employeeEmail", "title": {"en": "Email" }, "type": "string" },
              { "key": "phoneNumber", "title": {"en": "Email" }, "type": "string" },
              { "key": "age", "title": {"en": "Email" }, "type": "string" }
            ],
            "prop": {
              "hideNameAndDescription": true
            }
          }

forms:
    - key: employeeForm
      ref: json-forms/employeeForm.json
